test/lib/ufe/testSceneSegment.py

```diff
@@ -126,5 +126,12 @@ def testFilteredFindGatewayItems(self):
         result = handler.findGatewayItems(proxyShapeParentPath, otherRunTimeId)
         self.assertTrue(result.empty())
 
+    def testRootSceneSegmentRootPath(self):
```

Comment:
You need to run this test only if the new method is available. I would suggest something like this:
`@unittest.skipUnless(hasattr(ufe.SceneSegmentHandler, "rootSceneSegmentRootPath"), "Test requires ufe.SceneSegmentHandler, "rootSceneSegmentRootPath")`